### PR TITLE
Created example configuration section.

### PR DESCRIPTION
**Pull request by kkasperczyk-no**

### CODE REVIEW

File: docs/guides/nrfconnect_examples_configuration.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Configuring nRF Connect SDK examples
+
+When you tried to build nRF Connect SDK example applications like
+[lock-app](../../examples/lock-app/nrfconnect/README.md),
+[lighting-app](../../examples/lighting-app/nrfconnect/README.md) or
+[pigweed-app](../../examples/pigweed-app/nrfconnect/README.md), you could notice
+that some default configuration is used then. Probably after testing an example
```
Comment (LuDuda) [inline docs/guides/nrfconnect_examples_configuration.md:7]: > Probably after testing an example solution you may want to change something

I think i would remove this sentence.

File: docs/guides/nrfconnect_examples_configuration.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+<hr>
+
+## Configuring application
+
+Changing default application configuration might be done in the many ways and
+you should choose the one that suits you, depending on your needs. You are
+allowed to change configuration temporarily, what will be useful for testing
+impact of changes on the application behavior, but if you want to develop own
+application, probably you will be interested in making permanent changes.
```
Comment (LuDuda) [inline docs/guides/nrfconnect_examples_configuration.md:22]: @greg-fer is that okay to use sentences like: "probably you will be interested"? I would use different form of sentence to inform about the possibility to make permanent changes.

File: docs/guides/nrfconnect_examples_configuration.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+<hr>
+
+### Temporary configuration
+
+Temporary solution is basing on editing zephyr/.config, which is storing all
```
Comment (LuDuda) [inline docs/guides/nrfconnect_examples_configuration.md:28]: zephyr/.config -> **zephyr/.config** etc?

File: docs/guides/nrfconnect_examples_configuration.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+configuration options from the whole application generated as a result of
+building process. After cleaning build and deleting this file, all changes are
+gone, so this is not possible to save changes permanently this way.
+
+First thing that needs to be done is bulding the application (see:
+[Building nRF Connect examples](TODO:) for more details), by typing following
+command in the example directory:
+
+        # <board_name> should be replaced with the actual target board name
```
Comment (LuDuda) [inline docs/guides/nrfconnect_examples_configuration.md:37]: can we add:  for example: **nrf52840dk_nrf52840**

File: docs/guides/nrfconnect_examples_configuration.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+after that press `Q` to save and quit.
+
+At this point configuration changes are applied to the output file and it can be
+flashed to the device.
+
+<hr>
+
+### Permanent configuration
+
+Permanent solution is basing on changing configuration Kconfig files used as
```
Comment (LuDuda) [inline docs/guides/nrfconnect_examples_configuration.md:56]: In this chapter, I'm lacking information on where is the KConfig and helper overlays. As well how to enable them e.g. release.conf.

**Review by LuDuda — APPROVED** [review]
Good work!

I would try to be more specific on which filed need to be modified by the user or how to use overlays.

File: docs/guides/nrfconnect_examples_configuration.md (hunk 5 above)
Comment (kkasperczyk-no) [reply docs/guides/nrfconnect_examples_configuration.md:56]: If it comes to ovelays - ok, I will add it. Did you mean prj.conf or Kconfig file? Prj.conf location is described below and Kconfig location doesn't seem to me something interesting for the user, as he/she doesn't need to edit it.

Comment (Damian-Nordic) [reply docs/guides/nrfconnect_examples_configuration.md:56]: Note that I described using release.conf overlay in the Building section of the examples.

File: docs/guides/nrfconnect_examples_configuration.md (hunk 2 above)
Comment (kkasperczyk-no) [reply docs/guides/nrfconnect_examples_configuration.md:22]: Done

File: docs/guides/nrfconnect_examples_configuration.md (hunk 1 above)
Comment (kkasperczyk-no) [reply docs/guides/nrfconnect_examples_configuration.md:7]: I didn't remove it, but I reformatted it to be more informative.

**Comment by greg-fer** [comment]
I think this PR can be closed, no? It's included in #41

**Pull request closed without merging**